tests/__init__.py

```diff
@@ -97,13 +97,16 @@ def create_files(path: Union[str, Path], *contents: dict[str, Any]) -> None:
 
 
 def catalog_files(
-    catalog_name: str, operator: str, other_files: Optional[dict[str, Any]] = None
+    catalog_name: str,
+    operator: str,
+    other_files: Optional[dict[str, Any]] = None,
+    content: Any = None,
 ) -> dict[str, Any]:
     operator_path = f"catalogs/{catalog_name}/{operator}"
-
+    catalog_content = content or {"foo": "bar"}
```

Comment:
Shouldn't the content be a list of documents? Since we have the information needed (the operator name), could you perhaps use a list with a dummy olm.package entry and an olm.channel entry here so that the generated catalog looks more like a real one? Pretty much the same thing we are doing for the annotations in the `bundle_files()` function...

Reply:
Sample catalog added - I only used tuple instead, because as `create_files()` is written, it will create multi-doc yaml only from tuple. We can add more content to the default catalog if needed - currently there is no use for it anywhere in the code/tests.